addons/fieldmanual/CfgHints.hpp

```diff
@@ -0,0 +1,381 @@
+#include "script_component.hpp"
+
+/*
+    Preferred Image Size = 256x256
+    Can Embed Images Into Description = 1024x512
+        - Shadow = '0'
+        - Size = '9'
+
+    // ----- Arguments (For Keybinds To Show) ----- //
+
+    // SELF INTERACT KEYBIND
+    "
+        private _localized = 'Self Interact Key';
+        if !(isNil {['ACE3 Common', 'ace_interact_menu_selfInteractKey'] call CBA_fnc_getKeybind}) then {
+            _localized = ['ACE3 Common', 'ace_interact_menu_selfInteractKey'] call CBA_fnc_getKeybind select 8 select 0 call CBA_fnc_localizeKey;
+        };
+        _localized
+    "
+
+    // INTERACT KEYBIND
+    "
+        private _localized = 'Interact Key';
+        if !(isNil {['ACE3 Common', 'ace_interact_menu_interactKey'] call CBA_fnc_getKeybind}) then {
+            _localized = ['ACE3 Common', 'ace_interact_menu_interactKey'] call CBA_fnc_getKeybind select 8 select 0 call CBA_fnc_localizeKey;
+        };
+        _localized
+    "
+
+    // MEDICAL MENU KEYBIND
+    "
+        private _localized = 'Medical Menu Key';
+        if !(isNil {['ACE3 Common', 'ace_medical_gui_openMedicalMenuKey'] call CBA_fnc_getKeybind}) then {
+            _localized = ['ACE3 Common', 'ace_medical_gui_openMedicalMenuKey'] call CBA_fnc_getKeybind select 8 select 0 call CBA_fnc_localizeKey;
+        };
+        _localized
+    "
+*/
+
+class CfgHints
+{    
+    class ACE_Items
+    {
+        displayName = CSTRING(Items_DisplayName);
+        category = "ACE_FieldManual";
+
+        class ACE_Items_Base
+        {
+            image = "\z\ace\addons\common\data\logo_ace3_ca.paa";
+            noImage = 1;
+        };
+        class ACE_ATragMX: ACE_Items_Base
+        {
+            logicalOrder = 1;
+            displayName = CSTRING(Items_ATragMX_DisplayName);
+            displayNameShort = CSTRING(Items_ATragMX_ShortName);
+            description = CSTRING(Items_ATragMX_Description);
+            arguments[] = {};
+            tip = "<a href='https://ace3.acemod.org/wiki/feature/atragmx'>ATragMX Wiki</a>";
+        };
```

Comment:
Can't you inherit the `arguments[] = {};` part from each Base?

Reply:
You can, but the issue arises when you want to add additional arguments in special cases. You can't use

`arguements[] += {"myvalue"}` current in inherited configs

Reply:
Set up base class argument inheritance. Will make one large argument array within each parent class.